### PR TITLE
Fix credscan issue

### PR DESCRIPTION
**Pull request by barbaravaldez**

### CODE REVIEW

**Review by llali — APPROVED** [review]
:shipit:

File: tests/test_special.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def setUpClass(cls):
             list(client.execute_query('CREATE FUNCTION {0}() \
                                        RETURNS TABLE AS RETURN (select 1 as number);'
                                       .format(cls.function)))
-            list(client.execute_query('CREATE LOGIN {0} WITH PASSWORD=\'yoloC123445!\''
+            list(client.execute_query('CREATE LOGIN {0} WITH PASSWORD=\'PLACEHOLDER\''
```
Comment (llali) [inline tests/test_special.py:36]: have you verified the tests? please let me know if you need help with that

**Review by udeeshagautam — APPROVED** [review]

File: tests/test_special.py (hunk 1 above)
Comment (barbaravaldez) [reply tests/test_special.py:36]: I couldn't run them locally

Comment (llali) [reply tests/test_special.py:36]: feel free to merge. I'll fix the tests

**Pull request merged**